portfolio/src/main/webapp/script.js

```diff
@@ -70,6 +70,17 @@ function getComments() {
         comElement.appendChild(createComElement(comment));
     })
   });
+  fetch('/create-comment').then(response => response.text()).then((create) => {
+    console.log(create);
+    if(create.length  == 4){ //returned true
```

Comment:
I'm always weary of magic constants... what is "== 4"?!

Reply:
That is equal to the number of characters in true. I wasn't sure on how to pass booleans, so i just checked the size and compared that way.